src/main/resources/db/changelog/changes/01-create-table-book.yaml

```diff
@@ -0,0 +1,38 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-table-book
+      author: vrudkovsky
+      changes:
+        - createTable:
+            tableName: book
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: title
+                  type: varchar(255)
+              - column:
+                  name: author
+                  type: varchar(255)
+              - column:
+                  name: isbn
+                  type: varchar(255)
+                  constraints:
+                    unique: true
+                    nullable: false
+              - column:
+                  name: price
+                  type: decimal
+                  constraints:
+                    nullable: false
+              - column:
+                  name: description
+                  type: varchar(255)
+              - column:
+                  name: cover_image
+                  type: varchar(255)
```

Comment:
don't forget to leave last line of the file empty
![image](https://github.com/vrudkovsky/OnlineBookStore/assets/49767049/79ebe56b-84ca-473f-a61c-f0b62c41da1a)